ASA-DIA-InteractiveSafetyGraphics/safety-eDISH
# V0.9.1 dev

Fixes #65. 

To test:
- Load test page
- change a cutpoint to a negative number (it should auto-correct to 0)
- change the axis display from linear to log (the 0-axis should automatically update to match the lowest value in the data)
